### ISSUE TITLE
Crash on close

### ISSUE THREAD

**Issue by tvannahl** [flathub/com.discordapp.Discord#17]
Discord behaves rather strange when closing it on my PC (Fedora 27/Gnome) as well as some others I know of (Fedora 27/KDE and ArchLinux/KDE). When I try to close Discord a new window is spawned without content and I have to close both of those windows.

After I did that discord will be closed via segmentation fault. I do assume this is due to missing dependencies. I do come to that conclusion, because I cannot duplicate this behaviour when using the tar.gz version.

I can replicate that problem every time I close discord.

```
tvn@monolith:~ $ coredumpctl info 14529
           PID: 14529 (discord)
           UID: 1000 (tvn)
           GID: 1000 (tvn)
        Signal: 11 (SEGV)
     Timestamp: Mon 2018-01-15 16:59:44 CET (14min ago)
  Command Line: /app/extra/discord --type=renderer --no-sandbox --primordial-pipe-token=C3FCA1FC076C4B18840B17D77219EDFD --lang=de --app-path=/app/extra/resources/app.asar --node-integration=true --background-colo
    Executable: /newroot/app/extra/discord
 Control Group: /user.slice/user-1000.slice/session-3.scope
          Unit: session-3.scope
         Slice: user-1000.slice
       Session: 3
     Owner UID: 1000 (tvn)
       Boot ID: 572b0dcb09e94924a8241186049be01d
    Machine ID: fa2db5f0452348b3a264d669f36e68a6
      Hostname: monolith
       Storage: /var/lib/systemd/coredump/core.discord.1000.572b0dcb09e94924a8241186049be01d.14529.1516031984000000.lz4
       Message: Process 14529 (discord) of user 1000 dumped core.
                
                Stack trace of thread 665:
                #0  0x00007f1cb19c28e0 n/a (/home/tvn/.var/app/com.discordapp.Discord/config/discord/0.0.4/modules/discord_utils/discord_utils.node)
```

**Comment by Cxpher** [comment]
I was able to replicate this previously every time.

Now it's intermittent.

**Comment by TingPing** [comment]
Discord updates itself so they happen outside of `flatpak`.

**Comment by Cxpher** [comment]
Hi TingPing,

Well the problem still happens but i found that when it does, i just
minimize or close to try and then use the icon to quit and it quits.

Ideally of course, it does not happen at all.

**Comment by tvannahl** [comment]
@Cxpher took me some time to try it again. But at least on my system nothing has changed.

I am uncertain what information I could provide to narrow the problem down.

**Comment by Ads20000** [comment]
@popey [reckons this is fixed on version 0.0.5 (snap revision 60)](https://github.com/snapcrafters/discord/issues/3#issuecomment-385726895), might be worth trying that? :)

**Comment by tvannahl** [comment]
I did the update just now and I am happy to confirm: I am no longer able to reproduce the problem using Discord 0.0.5.
